prov/efa/src/rxr/rxr.h

```diff
@@ -645,10 +642,10 @@ void rxr_cq_complete_rx(struct rxr_ep *ep,
 			bool post_ctrl, int ctrl_type);
 
 void rxr_cq_write_tx_completion(struct rxr_ep *ep,
-				struct rxr_tx_entry *tx_entry);
+				struct rxr_op_entry *op_entry);
 
 void rxr_cq_handle_tx_completion(struct rxr_ep *ep,
```

Comment:
this function should be renamed to `handle_send_completion`.

Reply:
How about also changing rxr_cq_write_tx_completion and efa_cntr_report_tx_completion to rxr_cq_write_send_completion and efa_cntr_report_send_completion?

Reply:
No. `tx` and `send` are different. tx can be `send`, `read` or `write`.

when we write tx completion, we always take a TX_entry as an argument.